src/posix/platform/netif.cpp

```diff
@@ -115,31 +240,185 @@ enum
     kICMPv6MLDv2RecordChangeToExcludeType = 3,
     kICMPv6MLDv2RecordChangeToIncludeType = 4,
 };
+#endif
 
 static const size_t kMaxIp6Size = 1536;
 
+static bool gLogPackets = false;
```

Comment:
I've added a macro in the next version.  I've also switched to using otDumpInfo.  I don't think we need it there or enabled in general, but I don't want to remove it because it is very useful in debugging interactions, especially on a new platform.